spec/services/integration_customers/create_or_update_service_spec.rb

```diff
@@ -81,6 +83,31 @@
         expect { service_call }.to have_enqueued_job(IntegrationCustomers::CreateJob)
       end
 
+      context 'with multiple integration customers' do
+        let(:integration_anrok) { create(:anrok_integration, organization:) }
+        let(:integration_customers) do
```

Comment:
What happens when somebody tries to add multiple netsuite customers at the same time? Should we add a validation to prevent that?

Reply:
We have `validates :customer_id, uniqueness: {scope: :type}` this validation on model level and DB level (index unique validation) so I think we are safe. From UI it is not possible to send multiple objects of the same type. From the API it is possible but it should result with error.